widget/entry.go

```diff
@@ -1083,6 +1112,13 @@ func (r *entryRenderer) Refresh() {
 	if r.entry.ActionItem != nil {
 		r.entry.ActionItem.Refresh()
 	}
+	if r.entry.Validator != nil {
+		if !r.entry.Focused() && r.entry.Text != "" && r.entry.validationError != nil {
+			r.line.FillColor = &color.NRGBA{0xf4, 0x43, 0x36, 0xff}
```

Comment:
nit: add a similar TODO comment here so the hardcoded colour is replaced in future